dragonchain/lib/faas.py

```diff
@@ -0,0 +1,54 @@
+import os
+import base64
+import json
+from typing import List, Dict, Optional, cast, Any
+
+import requests
+
+from dragonchain import exceptions
+
+FAAS_GATEWAY = os.environ.get("FAAS_LOGS_GATEWAY") or os.environ["FAAS_GATEWAY"]
+
+
+def get_faas_auth() -> str:
+    """Gets authorization to use OpenFaaS
+
+    Returns:
+        A string containing authorization for OpenFaaS.
+        Returns empty string if authorization is not found, in the case of managed services
```

Comment:
I wouldn't say "in the case of managed services"

I would just say something like "if openfaas secrets aren't configured"